### DISCUSSION TITLE
How to show the Yuv/RGB color immediately

### DISCUSSION THREAD

**Discussion by milliyang** [IENT/YUView#394] [Q&A] [answered]
[Feature Request]:

when the mouse move over the image

show the Yuv/RGB/HSL color infomation on UI

I known,  by zoom in, we can see all the values, 

but,  it's not a convinent ways, when you has to zoom in, zoom out too many many times.

**Comment by ChristianFeldmann** [comment, marked as answer]
You can enable the `zoom box` (right click on the image and select `zoom box`). Now the underlying YUV or RGB values as well as the x/y pixel position will be shown. Also differences show detailed information and for some bitstreams you even get block info like prediction modes, motion vectors etc.
